recipes/qbs/all/conanfile.py

```diff
@@ -0,0 +1,83 @@
+from conans import ConanFile, tools
+import os
+
+
+class QbsConan(ConanFile):
+    name = "qbs"
+    description = "The Qbs build system"
+    topics = ("conan", "qbs", "build", "automation")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://qbs.io"
+    license = "LGPL-2.1-only", "LGPL-3.0-only", "Nokia-Qt-exception-1.1"
+    settings = {
+        "arch_build": ["x86_64"],
+        "os_build": ["Linux", "Macos", "Windows"]
```

Comment:
```suggestion
        "arch": ["x86_64"],
        "os": ["Linux", "Macos", "Windows"]
```
The new cross building support of conan will map `os` to `os_build` if used as a build requirement.

Reply:
Done.